aks-kubesphere-linux/README.md

```diff
@@ -0,0 +1,115 @@
+# Install KubeSphere on Azure Kubernetes Cluster
+
+> English | [中文](README_zh.md)
+
+Automatic deploy KubeSphere on Azure Kubernetes Cluster.
+Supported Kubernetes versions include `1.15.x`,`1.16.x`,`1.17.x`,`1.18.x`,`1.19.x`
+
+## Requirements
+1. Install Azure CLI
+[Install Azure CLI on Windows](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-windows?tabs=azure-cli)
+[Install Azure CLI on macOS](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-macos)
+
+2. Sign in
+[Sign in with Azure CLI](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli)
+
+3. Create a service principal
```

Comment:
can use use MSI for this sample?

Reply:
Is so cool, but it failed when my test, I will re-verify this soon.

Reply:
What was the failure?